open_rfps/Exchange Listing Activities

```diff
@@ -0,0 +1,27 @@
+Proposal due date: 19th of April 2022
+Proposal overview: This proposal regards the minting of XOR for activities pertaining to listings on Exchanges.
+Proposal goals: The goal is to list XOR PSWAP VAL and XSTUSD with a SORA network integration on a prime centralized exchange and support all subsequent activities, such as, marketing, etc.
+
+As the intended exchange is among the top 10  highest daily spot trading volume (at time of writing this proposal), it will be a good entry point for consideration of other major exchanges such as Binance or Kraken, as well as improving the current volume and growth rate of XOR, PSWAP, VAL and XSTUSD.
+
+Scope of work: 
+- Proposal approval to mint XOR as a loan (!)  to cover the necessary costs associated to listing in a centralized exchange
+- payment and execution of all listing related activities - due to contract clauses and NDA’s we can’t disclose the details of the pending agreements.
+
+Current roadblocks and barriers to success:
+Timeline of full payback is unclear, we aim at 6-9 months. there will be some selling pressure for XOR in the pre-listing phase (up to 1 month).
+We can’t disclose the details of our pending agreements
+
+Evaluation metrics and criteria: 
+Listing on centralized exchange with +$2B daily volume
+increased volume for all SORA tokens
+integration of SORA network
+More holders of our ecosystem tokens
+
+Submission requirements: 
+Collaboration by Soramitsu to integrate SORA network 
+Add budget to actual proposal
+
```

Comment:
Budget:
Mint XOR to get:
$141k XOR
$375k USDT
$85k ETH
$56k VAL
$50k XSTUSD
= $707k
(PSWAP will be provided by Polkaswap Community members, no mint required)